.github/workflows/update_goldens.yaml

```
@@ -0,0 +1,46 @@
+name: update_golden_tests
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)